pvr.vuplus/resources/language/resource.language.en_gb/strings.po

```diff
@@ -832,7 +832,7 @@ msgstr ""
 
 #help: Connection - connectionchecktimeout
 msgctxt "#30610"
-msgid "The value in seconds to wait for a connection check to complete before failure. Useful for tuning on older Enigma2 devices."
+msgid "The value in seconds to wait for a connection check to complete before failure. Useful for tuning on older Enigma2 devices. Note, this setting should rarely need to be changed. It's more likely the interval below will have the desired effect."
```

Comment:
I would always be very careful with statements like 'below' for describing the relation between UI elements. Who guarantees that the settings will be rendered by the UI top-down as specified in settings xml file? And if somebody changes the order of settings in the xml file later, I bet they forget to change this string.

Reply:
Ah, missed that thanks